rwgame/main.cpp

```diff
@@ -1,9 +1,23 @@
 #include "RWGame.hpp"
+#include "SDL.h"
+#include <iostream>
 
-int main(int argc, char *argv[])
+int main(int argc, char* argv[])
 {
+	try {
+		RWGame game(argc, argv);
 
-	RWGame game( argc, argv );
+		return game.run();
+	} catch (std::exception& ex) {
+		const char* kErrorTitle = "Fatal Error";
 
-	return game.run();
+		std::cerr << kErrorTitle << "\n" << ex.what() << std::endl;
```

Comment:
I feel like this should go through our "logger" system.
It should probably have a "fatal" option which forwards it to a callback or something which would be the SDL error box and controlled exit?

(Might be beyond the scope of PR)